src/Pages/Detail/Facility.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import "./Facility.scss";
+
+class Facility extends Component {
+  render() {
+    return (
+      <div className="Facility">
+        <div className="facilityInfoBox">
+          <h4 className="facilityHeader">시설 안내</h4>
+          <ol>
+            <li className="facilityList">
+              <span className="numbered">1</span>
+              <span className="dataInfo">빔프로젝터</span>
+            </li>
+            <li className="facilityList">
+              <span className="numbered">2</span>
+              <span className="dataInfo">음향시설</span>
+            </li>
+            <li className="facilityList">
+              <span className="numbered">3</span>
+              <span className="dataInfo">조명시설[색상조정가능]</span>
+            </li>
+            <li className="facilityList">
+              <span className="numbered">4</span>
+              <span className="dataInfo">냉장고 및 전기쿡탁</span>
+            </li>
+            <li className="facilityList">
+              <span className="numbered">5</span>
+              <span className="dataInfo">플레이트 및 커트러리</span>
+            </li>
+            <li className="facilityList">
+              <span className="numbered">6</span>
+              <span className="dataInfo">테이블 및 의자[최대10인]</span>
+            </li>
+            <li className="facilityList">
+              <span className="numbered">7</span>
+              <span className="dataInfo">에스프레소머신</span>
+            </li>
```

Comment:
이 부분 꼭 제대로 공부해주세요!!!!

이렇게 똑같이 반복되는 코드는 map으로 간단하게 줄일 수 있습니다!
변하는 데이터 부분을 배열로 만들어서 map 사용하시면 돼요!
그리고 이 데이터 변수는 나중에는 백엔드에서 받아오지만 우선 지금 사용하실 때는 컴포넌트 바깥에서 따로 선언해주시면 돼요!
만약에 render 안에서 이 변수를 선언하면 render 될 때마다 변수가 선언돼서 비효율적입니다!
그리고 map 사용할 때는 항상 제일 바깥 태그에 key를 넣어줘야 합니다.
이 key 값은 절대 겹치지 않을 값을 넣어줘야 해서
아래 예시 같은 경우에는 데이터에 있는 Id를 넣어줬어요!

```suggestion
const FACILITY_DATA = [{id: "1", name: "빔프로젝트"}, {id: "2", name: "음향시설"}, ....]
{FACILITY_DATA.map(el => {return (
      <li className="facilityList" key={el.id}>
              <span className="numbered">{el.id}</span>
              <span className="dataInfo">{el.name}</span>
            </li>
)}}
      
```

Reply:
넵~